README.md

```diff
@@ -108,10 +108,12 @@ To use this package in combination with Leaf to protect forms, there is a tag pr
 * Add `CSRFFormFieldTag` in `configure.swift`
 
 ```swift
-var tags = LeafTagConfig.default()
-// [...]
-tags.use(CSRFFormFieldTag.self, as "csrfFormField")
-services.register(tags)
+services.register { container -> LeafTagConfig in
+	var config = LeafTagConfig.default()
+	// [...]
```

Comment:
Well if one registers multiple tags, they should be configured there too, Right? So not  techically necessary, no.

Reply:
Well, actually it doesn't need to be there. I just think it's quite helpful to point out that this block is used in other situations as well.

Reply:
Since it's a matter of taste, I'd leave this decision to you. I prefer having this comment, but wouldn't shed a tear.

Reply:
I don’t mind comments. :) I guess I’m suggesting that the comment be more descriptive and informative.

Reply:
I guess you're right. With further thinking, I can't come up with a good descriptive comment. And I think it's probably not that useful like I thought it would be. So I guess I'll just remove it. :)

Reply:
Haha, just found a few lines above in the Middleware-section the line "// ..." - that's just the same here. :-) I'll match them.